<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioMenu</class>
 <widget class="QMainWindow" name="AudioMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>58</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLineEdit" name="samplerate_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="samplerate_label">
         <property name="text">
          <string>Частота дискретизации</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLineEdit" name="duration_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="duration_label">
         <property name="text">
          <string>Длительность (секунды)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLineEdit" name="decay_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="decay_label">
         <property name="text">
          <string>Коэффициент затухания</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>110</number>
       </property>
       <property name="rightMargin">
        <number>75</number>
       </property>
       <item>
        <widget class="QLineEdit" name="freqs_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="freqs_label">
         <property name="text">
          <string>Частоты</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>70</number>
       </property>
       <property name="rightMargin">
        <number>70</number>
       </property>
       <item>
        <widget class="QPushButton" name="write_button">
         <property name="text">
          <string>Записать звук в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
